packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -256,8 +256,8 @@ test.serial('makes usdc advance', async t => {
   harness?.resetRunPolicy();
 
   const doc = {
-    node: `fastUsdc.status`,
-    owner: `the statuses of fast USDC transfers identified by their tx hashes`,
+    node: `fastUsdc.txns`,
+    owner: `the Ethereum transactions upon which Fast USDC is acting`,
```

Comment:
these vstorage snapshots are nice for looking at an example or two, but I wonder about how to make it easier to figure out all possible values.

How about including the name of a TypedPattern and the Type in the `doc`?

Maybe `documentStorageSchema` should support the pattern?